_config.yml

```diff
@@ -2,13 +2,26 @@ title: Introduction to Software Development Tooling
 description: >
   Learn the command line, version control, build systems, and correctness in
   Tufts' CS 50ISDT.
-theme: jekyll-theme-primer
 google_analytics: UA-26440738-1
-url: 'https://bernsteinbear.com'
+url: "https://bernsteinbear.com"
 baseurl: /isdt
+favicon: /favicon.ico # TODO: verify
 permalink: pretty
 exclude:
   - README.md
   - LICENSE
   - Gemfile
   - Gemfile.lock
+
+# Configuration for Primer Spec
+# https://github.com/eecs485staff/primer-spec
+remote_theme: eecs485staff/primer-spec
```

Comment:
Can you pin this to a tag or branch in your repository? Right now it tracks a moving target and might cause our site to change unexpectedly without any explicit action from us.

Reply:
Great question! Primer Spec uses [Semantic versioning](https://github.com/eecs485staff/primer-spec/blob/main/docs/CONTRIBUTING.md#bumping-the-version-in-pull-requests), and I usually only release minor version bumps between semesters at the University of Michigan.

**Option 1:** It seems like the [Tufts semester schedule](https://students.tufts.edu/registrar/what-we-assist/course-registration-and-scheduling/academic-calendars) is somewhat similar to that of [UMich](https://ro.umich.edu/calendars). If you'd like, I can ensure that I do not make version bumps in the middle of a Tufts semester.

**Option 2:** You can [pin to a specific *minor* version of Primer Spec](https://github.com/eecs485staff/primer-spec/blob/main/docs/USAGE_ADVANCED.md#pinning-to-a-specific-version). You will always receive automatic `patch` version updates (usually minor changes to JS, or very minor CSS fixes).